go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/dustin/go-humanize v1.0.0
 	github.com/elgris/jsondiff v0.0.0-20160530203242-765b5c24c302
+	github.com/evanphx/json-patch v0.5.2 // indirect
```

Comment:
I'm not using this one, only the one below. I'm not sure why `go mod tidy` didn't remove it but it's not important. Will clean up in next pass.